website/source/docs/job-specification/task.html.md

```diff
@@ -66,6 +66,9 @@ job "docs" {
   defaults to the same user as the Nomad client. This can only be set on Linux
   platforms.
 
+- `vault` <code>([Vault][]: nil)</code> - Specifies the configuration options
```

Comment:
"Specifies the configuration options for connecting the tasks in this group to Vault." ->
Specifies the set of Vault policies required by the task. Overrides any `vault` block set at the `group` or `job` level.